common/sgx/tcbinfo.h

```diff
@@ -86,6 +86,24 @@ typedef struct _oe_parsed_tcb_info
 
 } oe_parsed_tcb_info_t;
 
+/*!
+ * Parse a SGX TCB status string to an OE TCB status.
+ *
+ * @param[in] str The SGX TCB status string to parse.
```

Comment:
OE style guidance is to avoid abbreviations, so "str" should be something like "status_string"